build/automation/lib/terraform.mk

```diff
@@ -3,7 +3,7 @@ TERRAFORM_DIR_REL = $(shell echo $(TERRAFORM_DIR) | sed "s;$(PROJECT_DIR);;g")
 TERRAFORM_STATE_KEY = $(PROJECT_GROUP_SHORT)-$(PROJECT_NAME_SHORT)/$(PROFILE)
 TERRAFORM_STATE_LOCK = $(or $(TEXAS_TERRAFORM_STATE_LOCK), terraform-service-state-lock-$(PROFILE))
 TERRAFORM_STATE_STORE = $(or $(TEXAS_TERRAFORM_STATE_STORE), terraform-service-state-store-$(PROFILE))
-TERRAFORM_VERSION = 0.12.29
+TERRAFORM_VERSION = 0.13.1
```

Comment:
This is a major version upgrade, so just a side note to myself to run a plan to make sure the current terraform configuration works with the latest version